DogServiceExample/pom.xml

```diff
@@ -51,29 +50,13 @@
       <version>1.3</version>
     </dependency>
 
-    <dependency>
-      <groupId>com.jayway.restassured</groupId>
-      <artifactId>rest-assured</artifactId>
-      <version>2.8.0</version>
-      <scope>test</scope>
-    </dependency>
 
     <dependency>
-      <groupId>com.jayway.restassured</groupId>
-      <artifactId>json-path</artifactId>
-      <version>2.8.0</version>
-    </dependency>
-
-    <dependency>
-      <groupId>com.jayway.restassured</groupId>
-      <artifactId>xml-path</artifactId>
-      <version>2.8.0</version>
-    </dependency>
-    <dependency>
       <groupId>com.google.code.gson</groupId>
       <artifactId>gson</artifactId>
       <version>2.2.4</version>
     </dependency>
+
     <dependency>
       <groupId>com.googlecode.json-simple</groupId>
       <artifactId>json-simple</artifactId>
```

Comment:
It's used in DogTest for assertion and @Test annotation... Substituted to:
    <dependency>
      <groupId>junit</groupId>
      <artifactId>junit</artifactId>
      <version>4.4</version>
      <scope>test</scope>
    </dependency>